kamaki/provision.py

```diff
@@ -0,0 +1,402 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from __future__ import (absolute_import, division,
+                        print_function, unicode_literals)
+import logging
+import argparse
+from cluster_error_constants import *
+from kamaki.clients import astakos, cyclades
+from kamaki.clients import ClientError
+from kamaki.clients.utils import https
+from kamaki.cli.config import Config as KamakiConfig
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+# TODO: remove this and actually use ssl cert files
+https.patch_ignore_ssl()
```

Comment:
please check this https://www.synnefo.org/docs/kamaki/latest/developers/clients-api.html#secure-connections